README.md

```diff
@@ -21,6 +21,23 @@ the content.
 The RukPak project is community driven and is part of the broader Kubernetes ecosystem. New contributors are welcome and
 highly encouraged. See the [contributing guidelines](CONTRIBUTING.md) to get started.
 
+## Quickstart
+
+The RukPak project consists of a series of controllers, known as provisioners, that install and manage content on a
+Kubernetes cluster. See below for a more detailed look into the APIs that RukPak provides.
```

Comment:
Might be worth linking to where this starts in the document so that a user can easily navigate to it.